dcmgr/spec/minimal_dcmgr.conf

```diff
@@ -3,4 +3,10 @@
 database_uri 'mysql2://localhost/wakame_dcmgr_test?user=root'
 amqp_server_uri 'amqp://localhost/'
 
+features {
+  openvnet true
+  vnet_endpoint "localhost"
+  vnet_endpoint_port 9090
+}
```

Comment:
Adding these here means it's not a minimal `dcmgr.conf` any more. Since this configuration is specific to your tests, it should either be a new `dcmgr.conf` file in your tests' sub-directory or you could write some code to define dcmgr.conf in a before block.

This code can be used to parse a string as dcmgr.conf

```ruby
def write_dcmgr_conf(conf)
  Dcmgr::Configurations.dcmgr.parse_dsl do |me|
    me.instance_eval(conf)
  end

  @dcmgr_conf_errors = []
  Dcmgr::Configuration.walk_tree(Dcmgr.Configurations.dcmgr) do |c|
    c.validate(@dcmgr_conf_errors)
  end
end
```

I didn't test this code so it might not work as is but it should be something like that. :)